hawk/src/main/java/com/orhanobut/hawk/Hawk.java

```diff
@@ -47,6 +51,10 @@ public static HawkBuilder init(Context context) {
       return remove(key);
     }
 
+    if (!isHawkInitialised()) {
+      return false;
```

Comment:
Please no exception. Maybe here the status could be used. If we use the status like "init_started", then the put operation could be tried later.
Therefore I suggest a status like "init_started". What do you think?